lms/djangoapps/teams/views.py

```diff
@@ -584,9 +584,21 @@ def delete(self, request, team_id):
         """DELETE /api/team/v0/teams/{team_id}"""
         team = get_object_or_404(CourseTeam, team_id=team_id)
         self.check_object_permissions(request, team)
+        memberships = list(CourseTeamMembership.get_memberships(team_id=team_id))
```

Comment:
Yeah, but by that time the `team.delete()` call on line 589 has changed the data behind the queryset to be a less-helpful empty list.